tests/contracts/cmp-buffer.clar

```diff
@@ -0,0 +1,31 @@
+(define-public (less-buffer)
+    (ok (< 0x0102 0x0103))
```

Comment:
Can we add at least one test with some buffers with different lengths?

Reply:
I added plenty of tests in 299cef6.

BTW, it should be easy to add property checks for these kind of tests also, I'll add an issue.